src/components/subject-homepage/subject-sidebar.tsx

```diff
@@ -21,49 +21,65 @@ const SubjectSidebar = (props: Props) => {
   const [isCollapsed, setIsCollapsed] = useState(false);
 
   return (
-    <div
-      className={cn(
-        "sticky top-0 hidden max-h-screen shrink-0 bg-primary-foreground px-3 pb-12 pt-[11.5rem] lg:block",
-        "overflow-y-auto overflow-x-hidden",
-        "transition-all duration-300 ease-in-out",
-        "scrollbar scrollbar-track-primary/10 scrollbar-thumb-primary/30 scrollbar-thumb-rounded-full scrollbar-w-1.5",
-        isCollapsed ? "w-16" : "w-72",
-      )}
-    >
-      <div className="flex items-center justify-between">
-        <h2
+    <div className="z-10">
+      <div className="fixed top-0 py-0 shadow sticky start-0 end-0 shadow-none">
```

Comment:
Conflicting classes here. VS Code will underline the conflicting ones. You should also use Prettier before committing which will also put conflicting classes next to each other.

Reply:
VS Code won't underline on its own. I had the extensions for underlining but they weren't active